config-service-impl/src/test/java/org/hypertrace/config/service/ConfigServiceGrpcImplTest.java

```diff
@@ -215,30 +218,21 @@ void deleteConfigs() throws IOException {
             .addConfigs(getConfigToDelete(context2))
             .build();
 
-    Map<ConfigResourceContext, Value> writeAllConfigsRequest =
-        Map.of(
-            getConfigResourceContext(context1),
-            emptyValue(),
-            getConfigResourceContext(context2),
-            emptyValue());
-
-    when(configStore.writeAllConfigs(writeAllConfigsRequest, ""))
+    when(configStore.getContextConfigs(
+            List.of(getConfigResourceContext(context1), getConfigResourceContext(context2))))
         .thenReturn(
-            List.of(
-                buildUpsertedConfig(context1, emptyValue(), config1, 10L, 20L),
-                buildUpsertedConfig(context2, emptyValue(), config2, 10L, 20L)));
+            Map.of(
+                getConfigResourceContext(context1),
+                    buildContextSpecificConfig(context1, config1, 10L, 20L),
+                getConfigResourceContext(context2),
+                    buildContextSpecificConfig(context2, config2, 10L, 20L)));
     Runnable runnable =
         () -> configServiceGrpc.deleteConfigs(deleteConfigsRequest, responseObserver);
     RequestContext.forTenantId(TENANT_ID).run(runnable);
-    verify(configStore, times(1)).writeAllConfigs(eq(writeAllConfigsRequest), eq(""));
-    verify(responseObserver, times(1))
-        .onNext(
-            DeleteConfigsResponse.newBuilder()
-                .addAllDeletedConfigs(
-                    List.of(
-                        buildContextSpecificConfig(context1, config1, 10L, 20L),
-                        buildContextSpecificConfig(context2, config2, 10L, 20L)))
-                .build());
+    verify(configStore, times(1))
+        .deleteConfigs(
+            eq(Set.of(getConfigResourceContext(context1), getConfigResourceContext(context2))));
+    verify(responseObserver, times(1)).onNext(any());
```

Comment:
Please restore the response verification. This shouldn't change if we haven't broken compat